pkg/kubernetes/scripts/policy.js

```diff
@@ -32,13 +32,51 @@
         '$rootScope',
         'kubeLoader',
         'kubeMethods',
-        function($q, $rootScope, loader, methods) {
+        'kubeSelect',
+        'KubeWatch',
+        'KubeRequest',
+        'KUBE_SCHEMA',
+        function($q, $rootScope, loader, methods, select, watch, KubeRequest, KUBE_SCHEMA) {
+
+            var apiGroup;
+            var RBAC_GROUP = "rbac.authorization.k8s.io";
+            var RBAC_API = "/apis/rbac.authorization.k8s.io/v1beta1";
```

Comment:
Is that "v1beta1" interface new in OpenShift 3.7? Will it keep that name after release, or is that going to need further adjustments?

Reply:
It's a pretty standard kubernetes version. At some point it will change to v1. But we should have several versions to figure out how we want to handle that. It will for sure be the version around in 3.7.